conda-forge.yml

```
@@ -10,4 +11,4 @@ github:
 provider:
   linux_aarch64: default
   linux_ppc64le: default
-test: native
+test: native_and_emulated
```

That should stay native as test are very slow for the emulated run
```suggestion
test: native
```